From a32c36d4ba6c66ef73fc5aeae759c97a88aa675e Mon Sep 17 00:00:00 2001
From: Stuart Menefy <stuart.menefy@codasip.com>
Date: Thu, 18 Jan 2024 13:58:07 +0000
Subject: [PATCH 06/17] hw/riscv/hobgoblin: Support two PLIC contexts

The Hobgoblin PLIC has been extended to support two contexts,
wired up to the machine and supervisor interrupts.

Signed-off-by: Stuart Menefy <stuart.menefy@codasip.com>
---
 hw/riscv/hobgoblin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/riscv/hobgoblin.c b/hw/riscv/hobgoblin.c
index 80e3ec257d..ef4f8ea2be 100644
--- a/hw/riscv/hobgoblin.c
+++ b/hw/riscv/hobgoblin.c
@@ -61,7 +61,7 @@ static const struct MemmapEntry {
     [HOBGOBLIN_DRAM] =     { 0x80000000, 0x40000000 },
 };
 
-#define HOBGOBLIN_PLIC_HART_CONFIG "M" /* Not "MS" unfortunatly */
+#define HOBGOBLIN_PLIC_HART_CONFIG "MS"
 #define HOBGOBLIN_PLIC_NUM_SOURCES 32
 #define HOBGOBLIN_PLIC_NUM_PRIORITIES 7
 #define HOBGOBLIN_PLIC_PRIORITY_BASE 0x0000
-- 
2.39.3

